musicbrainzngs/mbxml.py

```diff
@@ -523,6 +525,9 @@ def parse_cdstub(cdstub):
 
     return result
 
+def parse_offset_list(ol):
```

Comment:
tests added in [test_mbxml_discid.py](https://github.com/JonnyJD/python-musicbrainzngs/blob/35222a08be8c8a817fdc86bd2c47c2e1c09063df/test/test_mbxml_discid.py)